fvcore/common/config.py

```diff
@@ -44,22 +46,25 @@ def load_yaml_with_base(filename: str, allow_unsafe: bool = False):
         Returns:
             (dict): the loaded yaml
         """
-        with open(filename, "r") as f:
```

Comment:
using fvcore's `PathManager.open` will work out of the box for http URLs.

Reply:
fixed 👍